pkg/operator/certrotation/metrics/monitor.go

```diff
@@ -0,0 +1,195 @@
+package metrics
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	"k8s.io/apimachinery/pkg/labels"
+	corev1listers "k8s.io/client-go/listers/core/v1"
+	"k8s.io/client-go/util/cert"
+	"k8s.io/klog"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/openshift/library-go/pkg/crypto"
+)
+
+const (
+	// CertificateManagedLabelName marks config map or secret as object that contains managed certificates.
+	// This groups all objects that store certs and allow easy query to get them all.
+	// Also any object with this label will be collected by prometheus and transformed to metrics.
+	// The value of this label should be set to "true".
+	CertificateManagedLabelName = "auth.openshift.io/certificate-managed"
```

Comment:
remove this.  You can query on exists: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#set-based-requirement